### PR TITLE
Update the Readme files

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

**Review by AnnaKondratova — APPROVED** [review]

File: Readme.md [hunk 1]
```diff
@@ -3,15 +3,40 @@
...
-- Standalone SelectBoxes
+# SelectBoxes for DevExtreme - How to implement standalone and in-Form cascading SelectBoxes
+
+This example demonstrates how to implement cascading SelectBoxes in the following scenarios:
+
+- Ыtandalone SelectBoxes
 - SelectBoxes in Form
 
-The main idea is to get a changed value from the first editor and filter the data source in the second editor so that it shows relevant items.
+When you select a value from the first SelectBox, the second SelectBoxloads the filtered values. In this example, when you select a state only the cities from this state appear in the second SelectBox.
```
Comment (dirk-pieterse) [inline Readme.md]: `the second SelectBoxloads the filtered values` Do you mean 'the second SelectBox loads the filtered values'?

```suggestion
When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only the cities from the selected state appear in the second SelectBox.
```

File: Readme.md [hunk 2]
```diff
@@ -3,15 +3,40 @@
...
+
+![Implement standalone and in-Form cascading SelectBoxes](devextreme-cascading-selectboxes.png)
+
+To implement this, get a [value](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#value) from the first editor and [filter](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filter) the [dataSource](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#dataSource) by this **value** in the second editor so it shows relevant items.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
To implement this, get the [value](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#value) from the first editor and [filter](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/Methods/#filter) the [dataSource](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#dataSource) by this **value** in the second editor.
```

`To implement this` What does **this** refer to?

File: Vue/README.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
-# vue-cascading-selectboxes
+# Vue
+
+Take a look at the following files of this example to see the required code:
```
Comment (dirk-pieterse) [inline Vue/README.md]: `the required code` Why 'required'?

**Review by dirk-pieterse — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]

File: Angular/README.md [hunk 4]
```diff
@@ -1,11 +1,10 @@
 # Angular
 
-Take a look at the following files of this example to see the required code: 
+Take a look at the following files to see the code of this example:
```
Comment (dirk-pieterse) [inline Angular/README.md]:
```suggestion
The following files contain the code of this example: 

```

File: React/README.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # React
 
-Take a look at the following files of this example to see the required code: 
+Take a look at the following file to see the code of this example:
```
Comment (dirk-pieterse) [inline React/README.md]:
```suggestion
The following file contains the code of this example:
```

File: Readme.md [hunk 6]
```diff
@@ -3,15 +3,40 @@
...
-- Standalone SelectBoxes
+# SelectBoxes for DevExtreme - How to implement standalone and in-Form cascading SelectBoxes
+
+This example demonstrates how to implement cascading SelectBoxes in the following scenarios:
+
+- Ыtandalone SelectBoxes
 - SelectBoxes in Form
 
-The main idea is to get a changed value from the first editor and filter the data source in the second editor so that it shows relevant items.
+When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only the cities from the selected state appear in the second SelectBox.
```
Comment (dirk-pieterse) [inline Readme.md]:
```suggestion
When you select a value from the first SelectBox, the second SelectBox loads the filtered values. In this example, only cities from the selected state appear in the second SelectBox.
```

File: Vue/README.md [hunk 7]
```diff
@@ -1,4 +1,8 @@
-# vue-cascading-selectboxes
+# Vue
+
+Take a look at the following file to see the code of this example:
```
Comment (dirk-pieterse) [inline Vue/README.md]:
```suggestion
The following file contains the code of this example: 
```

File: NetCore/Readme.md [hunk 8]
```diff
@@ -2,7 +2,7 @@
 
 Take a look at the following files of this example to see the required code:
```
Comment (dirk-pieterse) [inline NetCore/Readme.md]:
```suggestion
The following files contain the code of this example: 
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**